docs/content/contributing/developer-guide/index.md

```diff
@@ -91,16 +93,19 @@ mvn clean install
     --8<-- "../src/test/resources/amazon-test-sample.properties"
     ```
 
-
-2. Run integration tests
+2. Run unit and S3 integration tests
    ```
-   mvn clean install -Pintegration-tests
+   mvn clean install -Punit-tests,it-s3
    ``` 
 
-
-### Docker
-
-TODO: Add a guide to run tests with MinIO and docker-compose
+3. If you have docker then you can also use `-Pit-minio` to run integration tests against MinIO
```

Comment:
Would be great to add that if an error occurs (container not found / can't start) the person should pull the image locally on his own `docker pull minio/minio:edge` (I had this issue someday)

Reply:
Thanks for the heads up! I've updated the docs to reflect this possible issue. :)

FYI I changed the `minio/minio` image to use a fixed version in your PR so we're not using the `edge` one. :)